docs/source/pep_generation.rst

```diff
@@ -6,18 +6,25 @@ PEP Page Generation
 Process Overview
 ----------------
 
-We are generating the PEP pages by lightly parsing and cleaning up the HTML output from the `PEP Mercurial Repository <http://hg.python.org/peps/>`_.
+We are generating the PEP pages by lightly parsing the HTML output from the
+`PEP Mercurial Repository <http://hg.python.org/peps/>`_ and then cleaning up
+some post-parsing formatting.
 
-The process is as follows:
+The PEP Page Generation process is as follows:
 
-1. Clone and/or update the PEP repo located in `settings.PEP_REPO_PATH` to the latest revision.
-2. Run `pep2html.py`
-3. Run `genpepindex.py`
-4. Once all PEP HTML pages are generated, run `./manage.py generate_pep_pages`
+1. Clone and/or update the PEP repo located in ``settings.PEP_REPO_PATH`` to
```

Comment:
It would be better to tell users to set the `PEP_REPO_PATH` setting in `pydotorg/settings/local.py`.